src/styles/blocks/variables.scss

```
@@ -0,0 +1,6 @@
+$daySize: 100px;
+$padding: 10px;
+$border: 1px;
+$gap: 1px;
+$calendarWidth: 7 * $daySize + $gap * 6;
+$dayNames: mon, tue, wed, thu, fri, sat, sun;
```

```suggestion
$dayNames: 
  mon 0, 
  tue ($block-size + $gap) * 2, 
```
you can add indents here and add them in loop